### PR TITLE
Floating action bar + item count + fix facets font size

### PR DESCRIPTION
**Pull request by EWhite613**
#patch#

# Changelog
* Floating action bar 
* Add selected item count 
* Set facets font size to medium

### CODE REVIEW

**Comment by EWhite613** [comment]
I extended the list to go to the bottom. Cause it looked dumb otherwise when only viewing 6
screen shot 2016-06-20 at 11 35 43 am

File: addon/styles/addon.scss [hunk 1]
```diff
@@ -29,14 +36,32 @@
...
+    background: rgba(255,255,255,0.95);
+    bottom: 0px;
+    box-shadow: 0 8px 8px -8px #696868, 0 -8px 8px -8px #696868;
+    position: absolute;
     .frost-button,
     .frost-link {
       margin-left: 5px;
     }
+    &::before{
+      content: counter(selected) " Items selected";
```
Comment (sglanzer) [inline addon/styles/addon.scss]: Any reason not to do this using 'selections' and the ember template?

**Comment by sglanzer** [comment]
Extending the list to the bottom is correct, so I'm glad that was done - I'm a little concerned that the percentage properties are going to get messed up in consuming apps, but we'll see

**Comment by sglanzer** [comment]
Are the buttons in the action bar fully right aligned?  If not, can we fix that?

**Comment by sglanzer** [comment]
I'm going to merge, but please raise follow up issues to address the comments

**Comment by sglanzer** [comment]
Approved

**Comment by EWhite613** [comment]
@sglanzer I fixed the positioning of the buttons in the commit after the pic

File: addon/styles/addon.scss (hunk 1 above)
Comment (EWhite613) [inline addon/styles/addon.scss]: @sglanzer No reason. I just like using Pure CSS when I can

Comment (sglanzer) [inline addon/styles/addon.scss]: Ok cool - for accessibility across devs I think it's good practice to adhere to standard styles of development when possible - definitely not a big deal, just a general note

**Pull request merged**